File: API/Features/Reservations/Dtos/PassengerReadDto.cs
using System;
using API.Infrastructure.Classes;

namespace API.Features.Reservations {

    public class PassengerReadDto {

        public int Id { get; set; }

        public Guid ReservationId { get; set; }

        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public string Remarks { get; set; }
        public string SpecialCare { get; set; }
        public bool IsCheckedIn { get; set; }

        public SimpleEntity Nationality { get; set; }
        public SimpleEntity Gender { get; set; }

    }

}

File: FrontEnd/src/app/features/reservations/classes/dtos/form/passenger-read-dto.ts
import { Guid } from 'guid-typescript'
// Custom
import { GenderActiveVM } from 'src/app/features/genders/classes/view-models/gender-active-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface PassengerReadDto {

    id: number
    reservationId: Guid
    gender: GenderActiveVM
    nationality: NationalityVM
    occupantId: number
    lastname: string
    firstname: string
    birthdate: string
    remarks: string
    specialCare: string
    isCheckedIn: boolean

}

export interface NationalityVM extends SimpleEntity { 
    
    code: string

}